homeassistant/components/tessie/entity.py

```diff
@@ -0,0 +1,47 @@
+"""Tessie parent entity class."""
+
+
+from typing import Any
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN, MODELS, TessieApi
+
+
+class TessieEntity(CoordinatorEntity):
```

Comment:
Now that you have a coordinator class you can make this coordinatorentity[yourcoordinator]